### DISCUSSION TITLE
USA + Canada - Combined M3U File

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#745] [Ideas]
Would it be possible to create a new USA and Canada combined m3u file link and make it available on your main page, 
as some channels are only available in the Canada m3u file and others are only available in the USA m3u file.

If you can create a USA and Canada combined M3U file, this would be extremely useful so we don't have to switch back and forth in m3u files to access our favorites. All channels inside mixed together, sorted by category.

Thanks so much for considering creating this new m3u file!

**Comment by what-a** [comment]
m3u4u is a free playlist editor which allows you to combine playlists:  http://m3u4u.com/
Start by reading through the Wiki:  https://wiki.m3u4u.com/en/home

The workflow within m3u4u would be:
"Create" USA playlist (using the IPTV-org URL)
"Create" Canada playlist (using the IPTV-org URL)
"Download" the merged USA/Canada playlist (use the "copy link" option rather than "download")
"Create" the merged USA/Canada playlist (using the link you copied from the instruction above)
Edit the merged USA/Canada playlist (e.g. sort alphabetically, hide certain groups, etc.)
"Download" the edited USA/Canada playlist (use the "copy link" option rather than "download")
Paste the m3u link in your IPTV player and bask in the glory of your creation.

Using URL links throughout rather than actually downloading files ensures that everything stays in sync going forward.

**Reply by brazenCoding** [reply]
I don't get how the playlists get merged from your description. I've created three playlists, one for each country above (and a third country) but your instructions jump to downloading a "merged playlist"?

**Comment by jphilippe2017** [comment]
Excellent reply, thank you! My only issue is that by doing this in the way you mentioned, will I not remove the regular IPTV-org updates?
I would like to keep the regular updates that are automatic if possible? This is why I made the suggestion of a new combined USA + Canada m3u link. What do you think? Will your way still provide the regular updates by IPTV-org?

**Reply by what-a** [reply]
Yes, using URL links (as noted in my instruction) will ensure that your merged list(s) receive regular updates from IPTV-org. 
This is possible because m3u4u pulls down fresh files, based upon the URL links you provided, once every 24 hours.
...also, make sure your IPTV player auto updates URL playlists.

**Comment by freearhey** [comment]
The "Northern America" region has been added to the list, which covers both countries:

```
https://iptv-org.github.io/iptv/regions/nam.m3u
```

**Comment by jphilippe2017** [comment]
Thank you, much appreciated!!

**Comment by maen-3Dexperimentssoftware** [comment]
What about Télémagino, Family Jr., etc.?
